SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -569,14 +569,10 @@
   <Target Name="AttachSonarQubeBuildWrapper" >
     <Message Importance="low" Text="Build wrapper binaries found in $(SQBuildWrapperBinPath)"/>
 
-    <PropertyGroup>
-      <SQBuildWrapperOutputPath Condition=" $(SQBuildWrapperOutputPath) == '' ">$(SonarQubeOutputPath)\bw\</SQBuildWrapperOutputPath>
-    </PropertyGroup>
-
     <MakeDir Directories="$(SQBuildWrapperOutputPath)" />
     <AttachBuildWrapper
+        AnalysisConfigDir="$(SonarQubeConfigPath)"
```

Comment:
Changed the targets to use the same approach as the _IsTestByName_ task - the config directory is passed in.